tests/rptest/scale_tests/many_topics_test.py

```diff
@@ -736,9 +816,13 @@ def _restart_unsafely(self):
         self._wait_workload_progress()
 
     def _isolate(self, nodes):
+        # if specific ports are blocked, like the port between brokers and clients,
+        # then the workload will not be able to progress. So disable that check in
+        # those cases.
+        non_progressing_ports = [9092]
         with firewall_blocked(nodes, self._target_port, full_block=True):
-            self.logger.info("Waiting for cluster to acknoledge isolation")
-            time.sleep(30)
+            self.logger.info("Waiting for cluster to acknowledge isolation")
+            time.sleep(10)
```

Comment:
why is this reduced to 10?

this smells flaky and is a good reason to try to use wait_until, see e.g., `is_node_isolated` and use elsewhere with `firewall_blocked`

Reply:
Honestly I didn't see the reason for waiting at all. `firewall_block` ssh's into each node then only returns when the port has been dropped from the ip tables and all sockets have been killed with `ss -K`. What is left to wait for at that point?

Reply:
yeah not sure, well fine as is I guess